src/Console/ModelsCommand.php

```diff
@@ -81,6 +82,7 @@ public function fire()
         $model = $this->argument('model');
         $ignore = $this->option('ignore');
         $this->reset = $this->option('reset');
+        $this->write_model_magic_where = $this->laravel['config']->get('ide-helper.write_model_magic_where');
```

Comment:
Can you change it here to set the default to true? Instead of in the property? In case the config isn't set. ` $this->laravel['config']->get('ide-helper.write_model_magic_where', true);`